services/BasicTypes.proto

```diff
@@ -99,6 +99,53 @@ message TokenTransferList {
     repeated NftTransfer nftTransfers = 3; // Applicable to tokens of type NON_FUNGIBLE_UNIQUE. Multiple list of NftTransfers, each of which has a sender and receiver account, including the serial number of the NFT
 }
 
+/* Representing the percentage of tokens to be transfers according to the TokenUnitFee*/
```

Comment:
Should it be `to be transferred` or `to transfer`?

Reply:
Fixed.